client/v3/client_test.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 func TestDialCancel(t *testing.T) {
-	defer testutil.AfterTest(t)
+	defer testutil.BeforeTest(t)
```

Comment:
Do we still keep `defer` when we call `BeforeTest`?

Reply:
Well spotted.  The only case I missed to remove defer... Wow.